src/Faithlife.Analyzers/Faithlife.Analyzers.csproj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.5.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.1.0" PrivateAssets="all" />
```

Comment:
IfNotNull analzyer fails on anything newer than 3.5.0.

Reply:
Sure enough. 🤦 Wasn't expecting any yak shaving, to be honest.